Monika After Story/game/script-topics.rpy

```diff
@@ -15724,3 +15724,58 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="mas_not_working",
```

Comment:
Better label name. Topics, especially repeatable ones, should always start with `monika_`
```suggestion
            eventlabel="monika_player_not_working",
```